### PR TITLE
Correct param values in lower or upper bound cases

### PR DESCRIPTION
**Pull request by dcherniev**
Corrected values for vrSynonyms and ttsName, that are sent in case when we should get their default values from lower or upper bound in json file.

### CODE REVIEW

**Comment by dcherniev** [comment]
Please review: @LuxoftAKutsan, @indlu, @VVeremjova, @LitvinenkoIra, @istoilovgithub

File: src/components/application_manager/src/application_manager_impl.cc [hunk 1]
```diff
@@ -2140,51 +2172,73 @@ void ApplicationManagerImpl::PullLanguagesInfo(const SmartObject& app_data,
...
   const SmartObject& languages = app_data[json::languages];
 
+  std::transform(cur_tts_lang.begin(),
+                 cur_tts_lang.end(),
+                 cur_tts_lang.begin(),
+                 ::toupper);
   std::transform(
       cur_vr_lang.begin(), cur_vr_lang.end(), cur_vr_lang.begin(), ::toupper);
 
+  ssize_t specific_tts_idx = -1;
```
Comment (istoilovgithub) [inline src/components/application_manager/src/application_manager_impl.cc]: @dcherniev It would be better to use std::npos here

Comment (dcherniev) [inline src/components/application_manager/src/application_manager_impl.cc]: @istoilovgithub You right, fixed, please check 18232e7

**Comment by dcherniev** [comment]
Rebuild required

**Comment by dcherniev** [comment]
Rebuild required

**Pull request merged**